clients/go/pkg/justification/processor.go

```diff
@@ -90,6 +92,23 @@ func (p *Processor) Process(ctx context.Context, logReq *api.AuditLogRequest) er
 	}
 
 	logReq.Payload.Metadata.Fields[LogMetadataKey] = structpb.NewStructValue(&tokStruct)
-	// TODO(#266): Also populate RequestAttribute.Reason.
+
+	justs, ok := (*tok).Get("justs")
+	if !ok {
+		logger.Warn("can't find 'justs' in claims")
```

Comment:
nit: try to avoid unnecessary nesting.
```suggestion
	if !ok {
		logger.Warn("can't find 'justs' in claims, skip logging RequestMetadata.RequestAttributes.Reason")
		return nil
	}
```

Reply:
To do this, we have to make sure the `reason` field is always that  last field we handle. In the future, if we need to populate other fields, it is easy to introduce a bug that we exit early.

Reply:
Fair enought